dev-qt/qt-creator/qt-creator-9999.ebuild

```diff
@@ -59,7 +59,7 @@ CDEPEND="
 	>=dev-qt/qtwidgets-${QT_PV}
 	>=dev-qt/qtx11extras-${QT_PV}
 	>=dev-qt/qtxml-${QT_PV}
-	clang? ( sys-devel/clang:8= )
+	clang? ( >=sys-devel/clang-8.0.1:= )
```

Comment:
This dependency would allow `sys-devel/clang:11` or newer.
To specify range of allowed slots of a multi-slotted package, `|| ( ... )` is needed.
When `:=` operator is needed, it should be used with `<` for minimal higher not allowed version (in this case `<sys-devel/clang-11:=`).
See e.g. `media-libs/mesa`: https://gitweb.gentoo.org/repo/gentoo.git/tree/media-libs/mesa/mesa-20.0.6.ebuild?id=dbf95d42221567866896f2e3b0ff14f719171724#n141
So in this case:
```
	clang? (
		|| (
			sys-devel/clang:10
			sys-devel/clang:9
			sys-devel/clang:8
		)
		<sys-devel/clang-$((LLVM_MAX_SLOT + 1)):=
	)
```